### PR TITLE
parameter R

### LINKED ISSUE #11: Parameter -R

### PR DESCRIPTION
**Pull request by justinmueller05082000**
fixes #11

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -92,9 +96,17 @@ func main() {
...
+				if bytes.Compare(dataSplit[j], dataSplit[j-1]) < 0 {
+					dataSplit[j], dataSplit[j-1] = dataSplit[j-1], dataSplit[j]
+					j -= 1
+				} else {
+					break
+				}
 			}
 		}
 	}
+
```
Comment (Al2Klimov) [inline main.go]: Not part of the task.

**Review by Al2Klimov — CHANGES_REQUESTED** [review]

**Review by Al2Klimov — CHANGES_REQUESTED** [review]
* [x] `git reset --soft HEAD~$(git rev-list --count $(git merge-base HEAD master)..HEAD^)`
* [x] `git commit --amend --no-edit`
* [x] `git push -f origin feature/parameter-R-11`

**Review by Al2Klimov — CHANGES_REQUESTED** [review]
Write a test protocol.

**Comment by justinmueller05082000** [comment]
**Test protocol:**

**toSort.txt**
```
c
d
U
H
a
j
a
B
J
G
R
A
b
T
s
t
```

**GO CODE:**
**Terminal:** `go run . -R toSort.txt` 
**Output:**
```
d
c
B
U
H
j
a
J
b
a
G
T
A
R
t
s
```

**SORT COMMAND:**
**Terminal:** `sort -R toSort.txt` 
**Output:**
```
T
a
a
b
U
s
R
B
d
G
H
j
c
J
t
A
```

**Conclusion:**
Both results are ordered randomly.

**Review by Al2Klimov — APPROVED** [review]

**Pull request merged**